### PR TITLE
Displays breadcrumbs in carrier settings form

### LINKED ISSUE #2050: Add breadcrumbs navigation to carrier accounts settings screen
As specified in Figma designs in #2028, breadcrumbs navigation must be added to the top of the screen.

Screen implemented as part of the work done in this PR: https://github.com/Automattic/woocommerce-services/pull/2049

### PR DESCRIPTION
**Pull request by Ferdev**
----

Closes #2050

To test it, just go to the UPS settings form screen. The breadcrumbs should be there:

`/wp-admin/admin.php?page=wc-settings&tab=shipping&section=woocommerce-services-settings`

### CODE REVIEW

File: classes/class-wc-connect-settings-pages.php [hunk 1]
```diff
@@ -89,9 +89,25 @@ public function output_shipping_settings_screen() {
...
+						$translated_carrier_name = __( 'UPS', 'woocommerce-services' );
+						break;
+					default:
+						break;
+				}
+				if ( $translated_carrier_name ) {
+				?>
+					<h2>
+						<a href="<?php echo esc_url( admin_url( 'admin.php?page=wc-settings&tab=shipping&section=woocommerce-services-settings' ) ); ?>"><?php esc_html_e( 'WooCommerce Services', 'woocommerce-services' ); ?></a> &gt;
+						<span class="wc-shipping-zone-name"><?php echo esc_html( $translated_carrier_name ); ?></span>
```
Comment (frosso) [inline classes/class-wc-connect-settings-pages.php:107]: (minor) I don't see a style assigned to `wc-shipping-zone-name` - is that going to be done in the future? Or can it be removed?

**Comment by Ferdev** [comment]
Hey @frosso sorry for getting back to you so late!

> Did you consider adding the breadcrumb to the view rendered by apps/shipping-settings, rather than adding the breadcrumb through JS? Was there a downside to that?

Actually, I just chose to do the same WC core was doing in the shipping settings page: I just copied the same breadcrumbs code they're using.

Yeah, as you have seen, there're several implementations for breadcrumbs in WordPress. I just decided to go with Core's approach.

File: classes/class-wc-connect-settings-pages.php (hunk 1 above)
Comment (Ferdev) [reply classes/class-wc-connect-settings-pages.php:107]: You're right, that class name shouldn't be there! I'll update the PR before merging it. Thanks for the review!! :bow:

**Comment by frosso** [comment]
@Ferdev thanks for following up! 💯  Looks good!

**Pull request merged**